Muehle_GUI.ipynb

```diff
@@ -220,6 +233,98 @@
     "    game_over = False"
    ]
   },
+  {
+   "source": [
+    "## Initialisierung und Funktionen der Status-Widgets und des Restar-Button"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def on_button_clicked(b):\n",
+    "    status.pieces= [[9,9],[          \n",
+    "                    [0,0,0,0,0,0,0,0], \n",
+    "                    [0,0,0,0,0,0,0,0], \n",
+    "                    [0,0,0,0,0,0,0,0]  \n",
+    "                   ]]\n",
+    "    status.current_player = 1\n",
+    "    status.mills = set()\n",
+    "    status.number_stones_to_remove = 0\n",
+    "    status.selected_dot_old = (None, None)\n",
+    "    status.selected_dot_new = (None, None)\n",
+    "    status.move_or_jump = False\n",
+    "    status.game_over = False\n",
+    "    update_board(status)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "turn_label = widgets.Label(value='Spieler x ist an der Reihe')\n",
+    "\n",
+    "status_label = widgets.Label(value='aktueller Status:')\n",
+    "\n",
+    "pieces_status_label = widgets.Label(value='Setzbare Steine:')\n",
+    "\n",
+    "restart_button = widgets.Button(description='restart')\n",
+    "restart_button.on_click(on_button_clicked)\n",
+    "\n",
+    "stones_player_one = widgets.FloatProgress(\n",
+    "                value=9,\n",
+    "                max=9,\n",
+    "                min=0,\n",
+    "                description='Spieler 1',\n",
+    "                bar_style='info', # 'success', 'info', 'warning', 'danger' or ''\n",
```

Comment:
Vielleicht kann man die auch an die Steinfarbe der Spieler anpassen

Reply:
Und richtig cool wäre es, wenn die Statusanzeige rechts neben dem Spielfeld ist. Dann hat man das schöner in einem Blickfeld.

Reply:
Steinfarben hab ich jetzt so gut wie möglich angepasst.